src/tools/structures/deleteMiseEnRelationStructureStatutAbandon.ts

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env node
+/* eslint-disable no-await-in-loop */
+
+// Lancement de ce script : ts-node src/tools/structures/deleteMiseEnRelationStructureStatutAbandon.ts
+
+import execute from '../utils';
+import service from '../../helpers/services';
+import { IMisesEnRelation } from '../../ts/interfaces/db.interfaces';
+
+execute(__filename, async ({ app, logger, exit }) => {
+  const misesEnRelation: IMisesEnRelation[] = await app
```

Comment:
Comme on peut avoir des pbs de cache, est-ce que ca n'aurait pas été + simple de cibler les structures en statut ABANDON puis les parcourir en faisant un deleteMany avec le match structure.$id & statut: { $in: ['nouvelle', 'nonInteressee', 'interessee'] }

Reply:
sauf que si on fait ça on va parcourir toutes les structures en ABANDON pour rien tout les jours

Reply:
ok faudra qu'on passe ne CRON la maj des caches dans l'ancien api je pense

Reply:
A faire comme pour les conseillers : partir collection structures $match abandon et updatedAt (voir pour echeck à -2 ?)